scripts/main.js

```diff
@@ -11,22 +11,41 @@ const bookTitle = getMyElement('#title-id');
 const bookAuthor = getMyElement('#author-id');
 const bookParent = getMyElement('.books');
 
-const bookArray = [];
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
 
-function Book(title, author) {
-  this.title = title;
-  this.author = author;
+  removeBook() {
+    const key = `${this.title} + ${this.author}`;
+    localStorage.removeItem(key);
+  }
 }
+/* eslint max-classes-per-file: ["error", 2] */
+
+class Books {
+  constructor() {
+    this.books = [];
+  }
+
+  addBook(title, author) {
+    const book = new Book(title, author);
+    this.books.push(book);
+    return book;
+  }
+}
+
+const newBook = new Books();
 
 function saveLocalStorage() {
   const key = `${bookTitle.value} + ${bookAuthor.value}`;
-  localStorage.setItem(key, JSON.stringify(bookArray));
+  localStorage.setItem(key, JSON.stringify(newBook));
 }
 
 function addBook(e) {
```

Comment:
- [ ] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.